libafl_frida/src/helper.rs

```diff
@@ -549,20 +547,24 @@ where
                     );
                 }
 
-                if let Some(_rt) = runtimes.match_first_type_mut::<DrCovRuntime>() {
-                    basic_block_size += instr_size;
-                }
+                basic_block_size += instr_size;
             }
             instruction.keep();
         }
         if basic_block_size != 0 {
+            log::trace!("{basic_block_start:#016X}:{basic_block_size:X}");
```

Comment:
Debugging code?

Reply:
As `log::trace` it's fine I would say (but it might need more information as to what is being traced)

Reply:
Removed